docs/_docs/sql-reference/explain-operators-list.adoc

```diff
@@ -0,0 +1,505 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= List Of Operators
+
+This section enumerates all operators with their semantic and supported attributes.
+
+== ColocatedHashAggregate
+
+The aggregate operation groups input data on one or more sets of grouping keys, calculating each aggregation function for each combination of grouping key.
+Colocated aggregate assumes that the data is already distributed according to grouping keys, therefore aggregation can be completed locally in a single pass.
+The hash aggregate operation maintains a hash table for each grouping set to coalesce equivalent tuples.
+The output rows are composed as follow: first come columns participated in grouping keys in the order they enumerated in `group` attribute, then come results of accumulators in the order they enumerated in `aggregation` attribute.
+
+Attributes:
+
+- `group`: Set of grouping columns.
+- `groupSets`: List of group key definitions for advanced grouping, like CUBE or ROLLUP.
```

Comment:
We should put all optional attributes after mandatory attributes to make this easier for users.